explorer/pages/_app.tsx

```diff
@@ -1,13 +1,15 @@
-import "../styles/globals.css";
-import type { AppProps } from "next/app";
+// Core dependencies
+import { theme } from "app/theme";
 import { ThemeProvider as Emotion10ThemeProvider } from "@emotion/react";
+import { CssBaseline } from "@mui/material";
 import { ThemeProvider } from "@mui/material/styles";
-import { theme } from "app/theme";
+import type { AppProps } from "next/app";
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <Emotion10ThemeProvider theme={theme}>
```

Comment:
Can you remind me why we thought both theme providers are necessary here?

Reply:
Let's revisit with [Revisit use of Emotion and MUI theme providers](https://github.com/clevercanary/data-browser/issues/78).